cypress/integration/accessibility.spec.js

```diff
@@ -1,95 +1,175 @@
 import { formatDate } from '../../src/utils/date';
 
 describe('Accessibility Requests', () => {
-  beforeEach(() => {
-    cy.localLogin({name: 'A11Y'});
-  });
-
-  it('can create a request and see its details', () => {
+  function create508Request(cy) {
     cy.visit('/508/requests/new');
-    cy.contains('h1', 'Request 508 testing')
-    cy.contains('label', "Choose the application you'd like to test")
+    cy.contains('h1', 'Request 508 testing');
+    cy.contains('label', "Choose the application you'd like to test");
     cy.get('#508Request-IntakeId')
       .type('TACO - 000000{enter}')
       .should('have.value', 'TACO - 000000');
     cy.contains('button', 'Send 508 testing request').click();
-    cy.location().should(loc => {
-      expect(loc.pathname).to.match(/\/508\/requests\/.{36}/)
-    })
-    cy.contains('li', 'Home')
-    cy.contains('li', 'TACO')
-    cy.contains('.usa-alert--success', '508 testing request created. We have sent you a confirmation email.')
-    cy.contains('h1', 'TACO')
-    cy.contains('h2', 'Next step: Provide your documents')
-    cy.contains('.usa-button', 'Upload a document')
+    cy.location().should((loc) => {
+      expect(loc.pathname).to.match(/\/508\/requests\/.{36}/);
+    });
+    cy.contains('li', 'Home');
+    cy.contains('li', 'TACO');
+    cy.contains(
+      '.usa-alert--success',
+      '508 testing request created. We have sent you a confirmation email.'
+    );
+    cy.contains('h1', 'TACO');
+    cy.contains('h2', 'Next step: Provide your documents');
+    cy.contains('.usa-button', 'Upload a document');
+  }
+
+  function addAndRemoveDocument(cy) {
+    cy.get('[data-testid="Upload-New-Document"]').click();
+    cy.contains('h1', 'Upload a document to');
+
+    // select document
+    cy.fixture('test.pdf').as('document');
+    cy.get('[data-testid="file-upload-input"]').then(function (el) {
+      const blob = Cypress.Blob.base64StringToBlob(
+        this.document,
+        'application/pdf'
+      );
+
+      const file = new File([blob], 'document.pdf', {
+        type: 'application/pdf',
+      });
+      const list = new DataTransfer();
+
+      list.items.add(file);
+      const myFileList = list.files;
+
+      el[0].files = myFileList;
+      el[0].dispatchEvent(new Event('change', { bubbles: true }));
+    });
+
+    // enter the document type
+    cy.contains('.usa-radio', 'Awarded VPAT')
+      .find('input')
+      .check({ force: true });
+
+    // click upload button
+    cy.get('[data-testid="Upload-Document"]').click();
+
+    cy.contains('.usa-alert', 'document.pdf uploaded to');
+    // verify that the document is in the list
+
+    cy.get('[data-testid="AccessibilityDocumentsList"] tbody tr')
+      .should('have.length', 1)
+      .first()
+      .within(() => {
+        cy.contains('Awarded VPAT');
+        cy.contains('Virus scan in progress...');
+      });
+
+    // view document
+    // remove document
```

Comment:
Is this a TODO?  Does document removal still need to be coded into this function?

Reply:
Hmm, do we not have removal functionality yet?  I was going by the ticket description, but I don't see a way to remove a document.

Reply:
Talking to myself here. . .I'm re-reading your description and seeing this is just the first half of the story. . .

Reply:
Yup! Those other steps require having a way to mark the files as having clean virus scans, and we don't have a way to do that from within Cypress tests yet.